core/templates/pages/landing-pages/topic-landing-page/topic-landing-page.controller.ts

```diff
@@ -104,15 +107,20 @@ angular.module('oppia').directive('topicLandingPage', [
           };
           ctrl.$onInit = function() {
             pathArray = WindowRef.nativeWindow.location.pathname.split('/');
-            ctrl.subject = pathArray[2];
-            topic = pathArray[3];
+            ctrl.subject = pathArray[1];
+            topic = pathArray[2];
             topicData = TOPIC_LANDING_PAGE_DATA[ctrl.subject][topic];
             landingPageData = topicData.page_data;
             ctrl.topicTitle = topicData.topic_title;
             ctrl.lessons = landingPageData.lessons;
             assetsPathFormat = '/landing/<subject>/<topic>/<file_name>';
-            var pageTitle = 'Learn ' + ctrl.topicTitle + ' - Oppia';
-            PageTitleService.setPageTitle(pageTitle);
+
+            ctrl.pageTitle = (
+              ctrl.topicTitle + ' | ' +
+              (typeof tagLines[ctrl.topicTitle] !== 'undefined' ?
```

Comment:
This looks very complicated, can we use the values of `TOPIC_LANDING_PAGE_DATA` to generate this easily? (Or maybe define this explicitly in `TOPIC_LANDING_PAGE_DATA`)

Reply:
I've defined this explicitly in `TOPIC_LANDING_PAGE_DATA`.